docker/Dockerfile

```diff
@@ -0,0 +1,36 @@
+FROM ubuntu:20.04
+
+ENV DEBIAN_FRONTEND=noninteractive
+ENV TZ=Europe/Berlin
+
+RUN apt update \
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Pin versions in apt get install. Instead of `apt-get install <package>` use `apt-get install <package>=<version>`](https://app.codacy.com/manual/dweindl/pyPESTO/pullRequest?prid=5388382)